src/commands/games/leagueoflegends/champion.js

```diff
@@ -5,6 +5,7 @@ module.exports = class LeagueOfLegendsChampion extends Command {
     super({
       name: 'champion',
       aliases: ['champ', 'c'],
+      category: 'games',
       parent: 'leagueoflegends',

```

Comment:
As you see, this is a subcommand (parent command is defined). This means that this subcommand already inherit from the parent one(in this case it's `leagueoflegends` command), so you don't need to specify it here

Reply:
Will fix this (and all the other subcommands I messed up)!